pkg/config/expand.go

```diff
@@ -585,19 +597,31 @@ func (dc *DeploymentConfig) expandVariables() {
 	}
 
 	for iGrp, grp := range dc.Config.DeploymentGroups {
-		for iMod := range grp.Modules {
+		for iMod, mod := range grp.Modules {
 			context := varContext{
 				groupIndex: iGrp,
 				modIndex:   iMod,
 				blueprint:  dc.Config,
 			}
 			err := updateVariables(
 				context,
-				dc.Config.DeploymentGroups[iGrp].Modules[iMod].Settings,
+				mod.Settings,
 				dc.ModuleToGroup)
 			if err != nil {
 				log.Fatalf("expandVariables: %v", err)
 			}
+
+			// ensure that variable references to projects in required APIs are expanded
+			for projectID, requiredAPIs := range mod.RequiredApis {
+				if isDeploymentVariable(projectID) {
+					s, err := updateVariableType(projectID, varContext{blueprint: dc.Config}, make(map[string]int))
```

Comment:
Since this should _always_ be a string, passing it into `handleVariable` should have the same effect.

Reply:
Done on HEAD